RefrigeratorExe/RefrigeratorExe/Program.cs

```diff
@@ -0,0 +1,297 @@
+﻿namespace RefrigeratorExe
+{
+    internal class Program
+    {
+       
+        static void Main(string[] args)
+        {
+
+            bool flag = true;
+            int numberPress;
+            List<Refrigerator> refrigerators = new List<Refrigerator>();
+            Refrigerator refrigerator = new Refrigerator("Samsong","Red",4);
```

Comment:
Usually the main function is small and short. So, even though you were not requested, you can create a RefrigiratorManager class which can initialize all the refrigerators and call their methods. And Program main will simply hold a single instance of  RefrigiratorManager